test/test_dtype_alu.py

```diff
@@ -60,24 +64,28 @@ class ht:
   bool = strat.booleans()
 
 def universal_test(a, b, dtype, op):
-  if not isinstance(op, tuple): op = (op, op)
+  if not isinstance(op, tuple): op = (op, op, op)
   tensor_value = (op[0](Tensor([a], dtype=dtype), Tensor([b], dtype=dtype))).numpy()
-  numpy_value = op[1](np.array([a]).astype(_to_np_dtype(dtype)), np.array([b]).astype(_to_np_dtype(dtype)))
-  if dtype is dtypes.bfloat16: np.testing.assert_allclose(tensor_value, numpy_value, atol=1e-3, rtol=1e-2)
-  elif dtype in dtypes_float: np.testing.assert_allclose(tensor_value, numpy_value, atol=1e-10)
-  else: np.testing.assert_equal(tensor_value, numpy_value)
+  if dtype in torch_dtypes_map.keys():
+    reference_value = op[2](torch.tensor([a]).to(torch_dtypes_map[dtype]), torch.tensor([b]).to(torch_dtypes_map[dtype])).float().numpy()
```

Comment:
use `.item()` to avoid the `float()` conversion should work on torch tensor

Reply:
Thanks